hips/draw/simple.py

```diff
@@ -178,11 +179,48 @@ def draw_tiles(self) -> np.ndarray:
 
         return image
 
+    def draw_hips_tile_grid(self) -> None:
+        """Draw lines on the output image (mainly used for debugging)."""
+        import matplotlib.pyplot as plt
+        for tile in self.tiles:
+            radec = tile.meta.skycoord_corners
```

Comment:
I don't think this will work if you have tiles in the "galactic" frame.
In that case, `tile.meta.skycoord_corners` will be a skycoord in the Galactic frame, i.e. not `radec` like you call the local variable.

There's several ways to handle this, to make it work for any combination of frames.
I would suggest you do
```
pos = tile.meta.skycoord_corners.transform_to(self.geometry.celestial_frame)
```
here, and then below pass `transform=ax.get_transform('world')`

Also, please make a notebook somewhere (not in this repo) and link to it, showing that this code works for two examples: galactic hips tiles drawn on galactic image, icrs hips tiles drawn on galactic image.